provision/provision-site.sh

```diff
@@ -41,7 +41,7 @@ get_config_value() {
 }
 
 get_hosts() {
-  local value=$(shyaml get-values "sites.${SITE_ESCAPED}.hosts" 2> /dev/null < ${VVV_CONFIG})
+  local value=$(shyaml get-values-0 "sites.${SITE_ESCAPED}.hosts" 2> /dev/null < ${VVV_CONFIG} | tr '\0' ' ' | sed 's/ *$//')
```

Comment:
Could this be adjusted instead to something like this?

```suggestion
  local hosts=$(shyaml get-values-0 "sites.${SITE_ESCAPED}.hosts" 2> /dev/null < ${VVV_CONFIG})
  # strip out terminator characters:
  value="${hosts//\0/}"
```

Reply:
I haven't tested the specific bash substitution used here, but it should run almost twice as fast as firing up `tr` and `sed`, and it's a lot easier to understand